cpufreq/src/cpufreq-monitor.c

```diff
@@ -186,61 +186,61 @@ cpufreq_monitor_set_property (GObject      *object,
                               const GValue *value,
                               GParamSpec   *spec)
 {
-        CPUFreqMonitor *monitor;
+    CPUFreqMonitor *monitor;
 
-        monitor = CPUFREQ_MONITOR (object);
+    monitor = CPUFREQ_MONITOR (object);
 
-        switch (prop_id) {
-        case PROP_CPU: {
-                guint cpu = g_value_get_uint (value);
+    switch (prop_id) {
+    case PROP_CPU: {
+        guint cpu = g_value_get_uint (value);
 
-                if (cpu != monitor->priv->cpu) {
-                        monitor->priv->cpu = cpu;
-                        monitor->priv->changed = TRUE;
-                }
+        if (cpu != monitor->priv->cpu) {
+            monitor->priv->cpu = cpu;
+            monitor->priv->changed = TRUE;
         }
-                break;
-        case PROP_ONLINE:
-                monitor->priv->online = g_value_get_boolean (value);
-
-                break;
-        case PROP_FREQUENCY: {
-                gint freq = g_value_get_int (value);
-
-                if (freq != monitor->priv->cur_freq) {
-                        monitor->priv->cur_freq = freq;
-                        monitor->priv->changed = TRUE;
-                }
+    }
+        break;
+    case PROP_ONLINE:
+        monitor->priv->online = g_value_get_boolean (value);
+
+        break;
+    case PROP_FREQUENCY: {
+        gint freq = g_value_get_int (value);
+
+        if (freq != monitor->priv->cur_freq) {
+            monitor->priv->cur_freq = freq;
+            monitor->priv->changed = TRUE;
         }
-                break;
-        case PROP_MAX_FREQUENCY: {
-                gint freq = g_value_get_int (value);
-
-                if (freq != monitor->priv->max_freq) {
-                        monitor->priv->max_freq = freq;
-                        monitor->priv->changed = TRUE;
-                }
+    }
+        break;
```

Comment:
break sentence should be within case scope

See AfterCaseLabel in https://clang.llvm.org/docs/ClangFormatStyleOptions.html

<img width="112" alt="Captura de Pantalla 2020-11-19 a les 22 17 07" src="https://user-images.githubusercontent.com/10171411/99725224-11bd5500-2ab5-11eb-803b-42294431e52c.png">

Reply:
Like this?
```
    switch (prop_id) {
    case PROP_CPU: {
        guint cpu = g_value_get_uint (value);

        if (cpu != monitor->priv->cpu) {
            monitor->priv->cpu = cpu;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    case PROP_ONLINE:
        monitor->priv->online = g_value_get_boolean (value);

        break;
    case PROP_FREQUENCY: {
        gint freq = g_value_get_int (value);

        if (freq != monitor->priv->cur_freq) {
            monitor->priv->cur_freq = freq;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    case PROP_MAX_FREQUENCY: {
        gint freq = g_value_get_int (value);

        if (freq != monitor->priv->max_freq) {
            monitor->priv->max_freq = freq;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    case PROP_GOVERNOR: {
        const gchar *gov = g_value_get_string (value);

        if (monitor->priv->governor) {
            if (g_ascii_strcasecmp (gov, monitor->priv->governor) != 0) {
                g_free (monitor->priv->governor);
                monitor->priv->governor = gov ? g_strdup (gov) : NULL;
                monitor->priv->changed = TRUE;
            }
        } else {
            monitor->priv->governor = gov ? g_strdup (gov) : NULL;
            monitor->priv->changed = TRUE;
        }
        break;
    }
    default:
                G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
        break;
    }
```
Then it needs to be fixed at other places too.

Reply:
Oh, the break after default can be removed.

Reply:
Yes. The curly brackets are required in a case-block to delimit the scope of the declared variables within the block, by doing it this way, it makes easier to read the switch body, as the separation of cases is visually clearer.

Reply:
Done.
I just updated `cpufreq-prefs.c`(pt.7), Can you please take a look again?